release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -407,16 +418,30 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-4-16-removed-features"]
 === Removed features
-//.APIs removed from Kubernetes 1.27
-//[cols="2,2,2",options="header",]
-//|===
-//|Resource |Removed API |Migrate to
 
-//|`CSIStorageCapacity`
-//|`storage.k8s.io/v1beta1`
-//|`storage.k8s.io/v1`
+[id="ocp-4-16-removed-kube-1-29-apis"]
+==== Beta APIs removed from Kubernetes 1.29
+
+Kubernetes 1.29 removed the following deprecated API, so you must migrate manifests and API clients to use the appropriate API version. For more information about migrating removed APIs, see the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29[Kubernetes documentation].
```

Comment:
Isn't this more than one deprecated API?

Reply:
Good point. They had it listed like it was just one, but in the end it is two. Will update!